.github/workflows/run-connector-tests-command.yml

```diff
@@ -25,13 +25,17 @@ on:
   push:
 
 jobs:
+
   call-connector-ci-tests:
     # Run always for 'workflow_dipatch' events.
     # Run for 'push' events on forks, but only if the fork defines "BYO Secrets".
     if: >
-      ${{ github.event_name == 'workflow_dispatch' ||
-      (github.event_name == 'push' && github.repository != 'airbytehq/airbyte' &&
-      (vars.GCP_PROJECT_ID != '')) }}
```

Comment:
TIL: The "${{ }}" wrapper does not work with multi-line string scalars (`>`) or (`|`). For these cases, we need to omit the "${{" wrapper or else the expression will evaluate to an (always truthy) plain-old string.